### PR TITLE
go.mod: update google/go-dap to 0.5.0

### PR DESCRIPTION
**Pull request by polinasok**
`make vendor` failed, I just used  `go mod vendor` + `go mod tidy` after changing the `go.mod` file

### CODE REVIEW

**Comment by polinasok** [comment]
cc @hyangah

**Review by aarzilli — APPROVED** [review]
LGTM

File: go.sum [hunk 1]
```diff
@@ -18,6 +18,8 @@ github.com/google/go-dap v0.3.0 h1:Dc4izN0u4VhZERYrz80f1PSEoDsVfdVmdM/W82CxzFk=
 github.com/google/go-dap v0.3.0/go.mod h1:5q8aYQFnHOAZEMP+6vmq25HKYAEwE+LF5yh7JKrrhSQ=
 github.com/google/go-dap v0.4.0 h1:bWSjcM9zp/jEFD4YbWERcHSed8vHbEdk0rmTvqgXDAs=
```
Comment (hyangah) [inline go.sum:19]: didn't go mod tidy clean these up?

**Review by hyangah — APPROVED** [review]
LGTM Thanks!

**Review by derekparker — APPROVED** [review]
LGTM

**Pull request merged**